OpenRA.Game/Activities/Activity.cs

```diff
@@ -220,16 +219,16 @@ internal void OnActorDisposeOuter(Actor self)
 
 		public virtual bool Cancel(Actor self, bool keepQueue = false)
```

Comment:
Is the return value even still used anywhere, or can we make this void again?

Reply:
It was used in two places: `Wait`/`WaitFor` and `TransformCrusherOnCrush`. I have rewritten `Wait` and `WaitFor` to not need it anymore and made `Activity.Cancel` return void. I am not sure about `TransformCrusherOnCrush`. I have removed the check for now because I'm not sure it serves any practical purpose, but it would be good if someone could look at it.

Reply:
The cancellation in `TransformCrusherOnCrush` is no more bogus than it used to be (in other words: relatively bogus, but not your fault here).